docs/source/conf.py

```diff
@@ -24,7 +24,7 @@
 
 project = "gammasim-tools"
 copyright = "2022, gammasim-tools developers"
-author = "Raul R Prado and Orel Gueta and Gernot Maier"
+author = "Raul R Prado and Orel Gueta and Gernot Maier and Victor Barbosa Martins"
```

Comment:
Really wondering at how many places we have our names. Seems to me that there must be a better way to do that and have one single list of authors. But good that you found that!

Reply:
agreed. we need to make it more general and be able to change in a single place, I will have a look at it later